perfdash/www/script.js

```diff
@@ -19,103 +19,104 @@ var app = angular.module('PerfDashApp', ['ngMaterial', 'chart.js']);
 var PerfDashApp = function(http, scope) {
     this.http = http;
     this.scope = scope;
-    this.selectedResource = "pods";
-    this.selectedVerb = "GET";
     this.testNames = [];
-    this.series = [ "Perc99", "Perc90", "Perc50" ];
-    this.allData = null;
 };
 
 PerfDashApp.prototype.onClick = function(data) {
     console.log(data);
-    window.location = "http://kubekins.dls.corp.google.com/job/kubernetes-e2e-gce-scalability/" + data[0].label + "/"
+    var job = document.getElementById("job").innerHTML
```

Comment:
Use `var job = this.job`. You've got the job property on the controller anyways. See index.html#37